//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// acde_NO_AP.h
//
// Code generation for function 'acde_NO_AP'
//

#ifndef ACDE_NO_AP_H
#define ACDE_NO_AP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acde_NO_AP(double P_init, double V_init, double A_init, double P_wayp,
                double A_wayp, double V_max, double J_max, double J_min,
                creal_T t[14]);

#endif
// End of code generation (acde_NO_AP.h)
